lib/trino-hive-formats/src/main/java/io/trino/hive/formats/DataSeekableInputStream.java

```diff
@@ -58,12 +58,12 @@
 
     private int bufferFill;
 
-    public DataSeekableInputStream(SeekableInputStream inputStream)
+    public DataSeekableInputStream(TrinoInputStream inputStream)
```

Comment:
No seekable

Reply:
I added another commit to rename `DataSeekableInputStream` to `TrinoDataInputStream`